docs/io/visualization/interaction_radius_plot.ipynb

```diff
@@ -0,0 +1,7286 @@
+{
```

Comment:
<ol><li>this should be subheading of Static plot - so heading level needs to be same as real packets mode</li><li>change to "broader or narrower" - one can pass num_bins &gt; number of shells, right?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>